cmd/statusd/main.go

```diff
@@ -106,6 +108,21 @@ func main() {
 		go startCollectingStats(interruptCh, backend.NodeManager())
 	}
 
+	// Sync blockchain and stop.
+	if *exitWhenSynced > -1 {
```

Comment:
I'd add a log entry there that tells that the node will just sync blockchain and then exit. I'd also probably extract this to a separate function to keep `main()` simpler.

```
if *exitWhenSynced >= 0 {
     syncAndStopNode(backend)
     return
}
```

Reply:
Yes, sync and exit stuff look like a fairly big block of code in `main()`. I'll do both.